swift_browser_ui_frontend/src/views/AddContainer.vue

```diff
@@ -1,61 +1,72 @@
 <template>
-  <div 
-    id="addview"
-    class="contents"
-  >
-    <h1 class="title is-3 addcontainerhead">
-      {{ 
-        create 
-          ? $t('message.container_ops.addContainer') 
-          : $t('message.container_ops.editContainer') + container
+  <c-card class="addContainer">
+    <h2 class="title is-3">
+      {{
+        create
+          ? $t("message.container_ops.addContainer")
+          : $t("message.container_ops.editContainer") + folderName
       }}
-    </h1>
-    <b-message type="is-info">
-      {{ $t('message.container_ops.norename') }}
-    </b-message>
-    <b-field
-      :label="$t('message.container_ops.containerName')"
-    >
-      <b-input 
-        v-model="container"
-        name="container"
-        aria-required="true"
-        :disabled="!create"
-      />
-    </b-field>
-    <b-field
-      :label="$t('message.tagName')"
-      :message="$t('message.tagMessage')"
-    >
-      <b-taginput
-        v-model="tags"
-        ellipsis
-        maxlength="20"
-        icon="label"
-        has-counter
-        rounded
-        type="is-primary"
-        :confirm-keys="taginputConfirmKeys"
-        :on-paste-separators="taginputConfirmKeys"
-      />
-    </b-field>
-
-    <b-field>
-      <b-button
-        type="is-primary"
-        @click="create ? createContainer () : updateContainer ()"
+    </h2>
+    <c-card-content>
+      <p class="info-text is-size-6">
+        {{ $t("message.container_ops.norename") }}
+      </p>
+      <b-field
+        custom-class="has-text-dark"
+        :label="$t('message.container_ops.containerName')"
+      >
+        <b-input
+          v-model="folderName"
+          name="foldername"
+          aria-required="true"
+          :disabled="!create"
+          data-testid="folder-name"
+        />
+      </b-field>
+      <b-field custom-class="has-text-dark" :label="$t('message.tagName')">
+        <b-taginput
+          v-model="tags"
+          ellipsis
+          maxlength="20"
+          has-counter
+          rounded
+          type="is-primary"
+          :placeholder="$t('message.tagPlaceholder')"
+          :confirm-keys="taginputConfirmKeys"
+          :on-paste-separators="taginputConfirmKeys"
+          data-testid="folder-tag"
+        />
+      </b-field>
+      <p class="info-text is-size-6">
+        {{ $t("message.container_ops.createdFolder") }}
+        <b>{{ $t("message.container_ops.myResearchProject") }}</b>
+      </p>
+      <c-link
+        :href="`https://my.csc.fi/myProjects/project/${currentProjectID}`"
+        underline
+        target="_blank"
+      >
+        View project members
```

Comment:
This is missing internationalisation. Example for Finnish translation: *Näytä projektin jäsenet*